src/cartesian/ticks/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { CartesianTickItem } from '../../util/types';
+
+/**
+ * Given an array and a number N, return a new array which contains every nTh
+ * element of the input array. For n below 1, an empty array is returned.
+ * @param {any[]} array An input array.
+ * @param {integer} n A number
+ * @returns {any[]} The result array of the same type as the input array.
+ */
+function getEveryNth<Type>(array: Type[], n: number): Type[] {
```

Comment:
This could be on it's own. Maybe in something like `src/util/ArrayUtils.ts`?

Reply:
Yeah this isn't 'tick' specific per se.

Also if we're going to move to different naming conventions for `utils` and for our files in general lets discuss.

We'll end up having files in `util/SomeUtilFile.ts` as well as places like `cartesian/ticks/utils`. Which pattern do we follow? imo we should stick to something consistent for ease of contribution

Reply:
I find that things should be as close to their usage as possible. If used all across the places, a top level utils folder makes sense. But if used only within CartesianAxis, I would argue that is where it should live.

Reply:
_Right now_ it's only used in CartesianAxis. But when the need for the functionality comes along outside of `tick` or `cartesian` it's much easier to miss it if it's hidden a few subfolders deep. (Which then leads to just duplicating some similar logic ad-hoc where and when it's needed)

Reply:
Fair enough. For consistency I will move it back to a shared utils.